content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -659,6 +668,45 @@ there is no automatic Job restart once the Job status is `type: Failed`.
 That is, the Job termination mechanisms activated with `.spec.activeDeadlineSeconds`
 and `.spec.backoffLimit` result in a permanent Job failure that requires manual intervention to resolve.
 
+### Terminal Job conditions
+
+A Job has two possible terminal states, it ends up either succeeded, or failed,
+and these states are reflected by the presence of the Job conditions `Complete`
+or `Failed`, respectively.
+
+The failure scenarios encompass:
+- the `.spec.backoffLimit`
+- the `.spec.activeDeadlineSeconds` is exceeded
+- the `.spec.backoffLimitPerIndex` is exceeded (see [Backoff limit per index](#backoff-limit-per-index))
+- the Pod matches the Job Pod Failure Policy rule with the `FailJob` action  (see more [Pod failure policy](#pod-failure-policy))
+
+The success scenarios encompass:
+- the `.spec.completions` is reached
+- the criteria specified by the Job Success Policy are met (see more [Success policy](#success-policy))
```

Comment:
"Job Success Policy" should be "Job success policy"?
"Success policy" should be "success policy"?

Reply:
```suggestion
- the criteria specified by the `.spec.successPolicy` are met (see more [Success policy](#success-policy))
```

Maybe, field name would be better. WDYT? @mimowo @tengqm

Reply:
using generic terms or field names are both okay to me.

Reply:
Cool, I use the field name in this case here.